pkg/image/utilities/utilities.go

```diff
@@ -541,15 +558,20 @@ if [[ $help ]]; then
     exit 0
 fi
 
+if [ ! -z "${source_registry}" ]; then
+    source_registry="${source_registry}/"
+fi
+
 pulled=""
 while IFS= read -r i; do
     [ -z "${i}" ] && continue
+    i="${source_registry}${i}"
```

Comment:
same comment above; assume no trailing slash

```suggestion
    i="${source_registry}/${i}"
```

Reply:
That will fail if the source registry is empty (i.e. Docker Hub).